spec/rubocop/cop/rails/unknown_env_spec.rb

```diff
@@ -92,6 +92,15 @@
       RUBY
     end
 
+    it 'does not mutate the cop config' do
```

Comment:
This test is basically the same as the one above. Would ya'll prefer if I merged it with the one above and change the test description to be: "accepts local as an environment name and does not mutate the cop config"?